README.md

```diff
@@ -66,6 +66,28 @@ func propertySink<S: SinkType>(sink: S)(signal: Signal<S.Element, NoError>) -> P
 func propertySink<S: SinkType>(sink: S)(producer: SignalProducer<S.Element, NoError>) -> PropertyOf<S>
 ```
 
+## UIKit Extensions
+
+##### `UIButton.rex_pressed`
+
+Flexible way to bind CocoaAction to the press of button. 
```

Comment:
Can you add another sentence that the button is automatically disabled while the download is in progress.